shaders/pbr-frag.glsl

```diff
@@ -147,7 +147,11 @@ vec3 getIBLContribution(PBRInfo pbrInputs, vec3 n, vec3 reflection)
     float mipCount = 9.0; // resolution of 512x512
     float lod = (pbrInputs.perceptualRoughness * mipCount);
     // retrieve a scale and bias to F0. See [1], Figure 3
+#ifdef USE_TEX_LOD
+    vec3 brdf = SRGBtoLINEAR(texture2DLodEXT(u_brdfLUT, vec2(pbrInputs.NdotV, 1.0 - pbrInputs.perceptualRoughness), 0)).rgb;
```

Comment:
This is a separate bug, not sure if you intended to address it as part of this PR.  Is there some better way to do this, maybe turn off mipmapping at the WebGL level, rather than just always asking for a particular mip level?

Reply:
Indeed. Removed from here and will be put into a separate PR.

Setting filtering to `GL_LINEAR` should solve the problem as well and sounds like a better solution.